api/v1alpha1/releaseplan_types.go

```diff
@@ -35,9 +35,9 @@ type ReleasePlanSpec struct {
 	// +required
 	Application string `json:"application"`
 
-	// Collectors is a list of data collectors to be executed as part of the release process
+	// Collectors contains all the information of the collectors to be executed as part of the release workflow
 	// +optional
-	Collectors []Collector `json:"collectors,omitempty"`
+	Collectors *Collectors `json:"collectors,omitempty"`
```

Comment:
this requires the RP to look like this to be able to be applied:

```
---
apiVersion: appstudio.redhat.com/v1alpha1
kind: ReleasePlan
metadata:
  labels:
    release.appstudio.openshift.io/auto-release: 'true'
    release.appstudio.openshift.io/standing-attribution: 'true'
  name: collect-rp
spec:
  application: rh-advisories
  collectors:
    items:
      - name: scott-test
        params:
          - name: scott
            value: tom
        timeout: 60
        type: scott-type
```

I don't think we want `items` there do we?

Reply:
We want to add a service account that is used by all the collectors so we had to add both keys, `items` and `serviceAccountName`.

Reply:
I think having this `items` key is really ugly and not intuitive as none of the lists we have in any other CRDs has you list items